lib/gems/pending/appliance_console/external_httpd_authentication.rb

```diff
@@ -79,11 +79,12 @@ def activate
     end
 
     def post_activation
-      say("\nRestarting sssd and httpd ...")
-      %w(sssd httpd).each { |service| LinuxAdmin::Service.new(service).restart }
+      say("\nRestarting httpd, if running ...")
+      httpd_service = LinuxAdmin::Service.new("httpd")
+      httpd_service.restart if httpd_service.running?
 
-      say("Configuring sssd to start upon reboots ...")
-      LinuxAdmin::Service.new("sssd").enable
+      say("Restarting sssd and configure it to start on reboots ...")
+      LinuxAdmin::Service.new("sssd").enable.restart
```

Comment:
Can we do .restart.enable instead ?
i.e. we may not want to enable it on reboots if it can't restart